nautobot/docs/user-guide/feature-guides/data-compliance.md

```diff
@@ -8,15 +8,16 @@ This is ideal for implementing some kind of business logic or standardization re
+`DataCompliance` objects are only created for the attribute `__all__` (to represent the overall status) and attributes that have at some point been invalid. As an example, suppose there is a `DataComplianceRule` that checks the `location_type` and `status` attributes of a Location object named Location A. When this rule is run for Location A, both attributes are valid, so the only `DataCompliance` object created would be for `__all__` with a value of valid. Then, suppose Location A's `location_type` attribute is edited in a way that makes it invalid. A new `DataCompliance` object would be created for `location_type` stating why it is invalid, and the `__all__` object would be updated to now be invalid. Then, if `location_type` is edited again to bring it back into compliance, the `DataCompliance` objects for `location_type` and `__all__` would be updated to be valid.
 
 Any `DataComplianceRule` class can have a `name` defined to provide a friendly name to be shown within in the UI. The `enforce` attribute can also be set to decide whether or not the `ComplianceError` caught in the `audit` method is raised again to the `clean` method, acting like a `ValidationError` wherever the original `full_clean` was called. Setting `enforce` to `True` changes the `DataComplianceRule` from a passive validation of data to an active enforcement of the logic within it.
 
-> **Note:** Individual rules implemented using the `DataComplianceRule` class are re-ran and re-validated when the target object is modified and saved, in addition to run ad-hoc using the `RunRegisteredDataComplianceRules` job. However, `DataCompliance` objects that are created by the job for the built-in validation rules when using the `Run built-in validation rules?` option will not update nor be re-validated until the job is explicitly ran once again.
->
-> For example, if a user fixes an object attribute that was incompliant with a built-in rule and then navigates to its `Data Compliance` tab, the object will still show as invalid for that built-in rule. This will remain the case until the job is ran again with the `Run built-in validation rules?` option checked.
+!!! note
+    Rules created using the `DataComplianceRule` class automatically re-run and re-validate whenever the target object is modified and saved. They can also be run manually using the `RunRegisteredDataComplianceRules` job.
+    However, `DataCompliance` objects generated by the `Run built-in validation rules?` option in the job do not update or re-validate automatically. They will only refresh if the job is explicitly run again.
+    For example, if a user fixes a non-compliant attribute on an object and checks the Data Compliance tab, the object will still appear as invalid for that built-in rule. This invalid status won't update until the job is rerun with the `Run built-in validation rules?` option enabled.
```

Comment:
This is confusing to me even on a re-read. :-) Any chance we can fix this behavior now that this is in core, or at least elaborate on the example with a concrete example of an `object`, `non-compliant attribute`, and `built-in rule` that would demonstrate this behavior?

Reply:
Rewrote it with a concrete example